repl/scala-2.11/src/main/scala/org/apache/spark/repl/SparkILoop.scala

```diff
@@ -84,6 +84,7 @@ class SparkILoop(in0: Option[BufferedReader], out: JPrintWriter)
     echo(welcomeMsg)
     echo("Type in expressions to have them evaluated.")
     echo("Type :help for more information.")
+    echo("Type ctrl + c to exit.")
```

Comment:
Thanks for your advice. I do not think everyone knows the way to exit the scala, especially for beginner, and the clear explanation is helpful to them.